src/main/kotlin/io/github/anki/anki/controller/model/dtos.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.anki.anki.controller.model
+import io.github.anki.anki.repository.mongodb.model.MongoCard
+import io.github.anki.anki.service.model.Card
+import org.bson.types.ObjectId
+
+
+class CardDto (
+    var parentCollectionId: String,
+    var cardKey: String,
+    var cardValue: String,
+) {
+
+    private var _id: String? = null
+
+    var id: String?
+        get() = _id
+        set(value) {
+            _id = value
+        }
+
+
+    fun toEntity(): Card {
```

Comment:
maybe let`s make interface for this methods?
i mean toEntity and toModel
the same for models in service and repository